src/node/esbuildService.ts

```diff
@@ -123,8 +123,10 @@ function printMessage(m: Message, code: string) {
         .slice(0, line - 1)
         .map((l) => l.length)
         .reduce((total, l) => total + l + 1, 0) + column
-    console.error(
```

Comment:
This is intentional, we need print code frame by console.

Reply:
Here is also need use `throw Error` instead of `console.error`.The `try catch` is not print the code frame error, it only handle the error with eval `require('@vue/compiler-dom').generateCodeFrame(code, offset, offset + 1)`

Reply:
> The try catch is not print the code frame error, it only handle the error with eval require('@vue/compiler-dom').generateCodeFrame(code, offset, offset + 1)

Yes. Please see these images of build result. The first image is the result before fixed. I point two problems.
1) The code frame is printed Incompletely.
2) Build terminate successfully.

So I fixed problems and the second image is the result of fixed.
1) The code frame is printed completely.
2) Build terminate with error(exit(1)).
![image](https://user-images.githubusercontent.com/63966451/90634213-50e3cc00-e262-11ea-980c-e9c9aabf4739.png)
![image](https://user-images.githubusercontent.com/63966451/90634229-56d9ad00-e262-11ea-816f-8e4eff539016.png)

Reply:
This error is already fixed...

Reply:
Yes. But if another error occurs, the error handling this PR implemented will improve visibility and safety.

Reply:
Is this redundant to just `try` and `throw`?